frontend/src/components/CustomApproval/Settings/components/common/ExprEditor/factor.ts

```diff
@@ -6,21 +6,56 @@ export const NumberFactorList = [
   "affected_rows",
   "level",
   "source",
+  "expiration_days",
+  "export_rows",
 ] as const;
 
 export const StringFactorList = [
   "environment_id", // using `environment.resource_id`
   "project_id", // using `project.resource_id`
   "database_name",
+  "table_name",
   "db_engine",
   "sql_type",
 ] as const;
 
 export const FactorList = {
-  DDL: uniq([...HighLevelFactorList, ...StringFactorList]),
-  DML: uniq([...HighLevelFactorList, ...NumberFactorList, ...StringFactorList]),
+  DDL: uniq(
+    without(
+      [...HighLevelFactorList, ...StringFactorList],
+      "expiration_days",
```

Comment:
Actually we don't save the number of days, but a timestamp.

<img width="272" alt="image" src="https://github.com/bytebase/bytebase/assets/24653555/6620d524-16a1-46e7-b59c-3bc5fa9e7d85">

Reply:
I consider that's somehow different with `request.time` since `request.time` is actually a timestamp of N days from now (aka. when the issue is created) on.
But here `expiration_days` is actually a constant value of N days.

Reply:
Product-wise, we should use days, while we should use expiration_duration underlying just in case we need something less than a day.